### ISSUE TITLE
Failure of Success Criterion 1.3.1 due to inserting non-decorative content by using :before and :after pseudo-elements and the 'content' property in CSS

### ISSUE THREAD

**Issue by ngreimel-lt** [magento/magento2#25977]
### Preconditions (*)

1. Magento 2.3.2 & 2.4-develop

### Steps to reproduce (*)

1. Run an accessibility scan of stock Magento 2.3.2 using PowerMapper > SortSite
2. Set Accessibility options to WCAG 2.1 AA, Section 508 Refresh (2017)
3. Run the report
4. Observe report results for styles.css

### Expected result (*)

1. Clean SortSite report, no accessibility issues with core Magento

### Actual result (*)

1. There are errors. Specifically, "Content inserted with CSS is not read by some screen readers, and not available to people who turn off style sheets." in /static/version.../frontend/luma/default/en_US/css/styles.css
Confirmed present in vanilla Magento 2.4-develop 

Content inserted with CSS is not read by some screen readers, and not available to people who turn off style sheets. | WCAG 2.0 A F87Section 508 (2017) A F87 | 1 pages
-- | -- | --
  | Move the content from CSS to the HTML page so that all users can see it. Impact on users:NVDA 2018.4 IE11 Windows 10 Reading: Text inserted by CSS content property not read.NVDA 2017.3 IE11 Windows 10 Reading: Text inserted by CSS content property not read.NVDA 2016.2 IE11 Windows 10 Reading: Text inserted by CSS content property not read.JAWS 2018.1811.2 IE11 Windows 10 Reading: Text inserted by CSS content property not read.JAWS 18.0.5038 IE11 Windows 10 Reading: Text inserted by CSS content property not read.JAWS 17.0.2619 IE11 Windows 10 Reading: Text inserted by CSS content property not read. |   |  
  | https://i-25977-2-4-develop.instances.magento-community.engineering/pub/static/version1575999647/frontend/Magento/luma/en_US/css/styles-m.css

**Comment by ngreimel-lt** [comment]
@magento give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @ngreimel-lt. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by ngreimel-lt** [comment]
@magento give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @ngreimel-lt. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by okorshenko** [comment]
@magento give me 2.4-develop instance

**Comment by magento-engcom-team** [comment]
Hi @okorshenko. Thank you for your request. I'm working on Magento 2.4-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @okorshenko, here is your [Magento instance](https://i-25977-2-4-develop.instances.magento-community.engineering).
Admin access: [https://i-25977-2-4-develop.instances.magento-community.engineering/admin](https://i-25977-2-4-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by ngreimel-lt** [comment]
Confirmed present in vanilla Magento 2.4-develop 

Content inserted with CSS is not read by some screen readers, and not available to people who turn off style sheets. | WCAG 2.0 A F87Section 508 (2017) A F87 | 1 pages
-- | -- | --
  | Move the content from CSS to the HTML page so that all users can see it. Impact on users:NVDA 2018.4 IE11 Windows 10 Reading: Text inserted by CSS content property not read.NVDA 2017.3 IE11 Windows 10 Reading: Text inserted by CSS content property not read.NVDA 2016.2 IE11 Windows 10 Reading: Text inserted by CSS content property not read.JAWS 2018.1811.2 IE11 Windows 10 Reading: Text inserted by CSS content property not read.JAWS 18.0.5038 IE11 Windows 10 Reading: Text inserted by CSS content property not read.JAWS 17.0.2619 IE11 Windows 10 Reading: Text inserted by CSS content property not read. |   |  
  | https://i-25977-2-4-develop.instances.magento-community.engineering/pub/static/version1575999647/frontend/Magento/luma/en_US/css/styles-m.css

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-Charlie 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-29728` were created

**Issue Available**: @engcom-Charlie, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by magento-engcom-team** [comment]
Hi @ngreimel-lt. 

Thank you for your report and collaboration!

The related internal Jira ticket `MC-29728` was closed as `Fixed`. 

The fix will be available with the upcoming `2.4.3` release.
